App/package.json

```diff
@@ -10,8 +10,20 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.27",
+    "@fortawesome/free-solid-svg-icons": "^5.12.1",
+    "@fortawesome/react-native-fontawesome": "^0.2.3",
+    "@react-native-community/masked-view": "^0.1.7",
+    "@react-navigation/native": "^5.1.0",
+    "@react-navigation/stack": "^5.2.1",
     "react": "16.9.0",
-    "react-native": "0.61.5"
+    "react-native": "0.61.5",
+    "react-native-fontawesome": "^7.0.0",
```

Comment:
Remover se não utilizar

Reply:
Removed react-native-fontawesome and fortawesome dependices from package.json